### DISCUSSION TITLE
Quantum Mini Linux Development Kit | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1038] [Q&A]
# Quantum Mini Linux Development Kit | Seeed Studio Wiki

Quantum Mini Linux Development Kit

https://wiki.seeedstudio.com/Quantum-Mini-Linux-Development-Kit/

**Comment by DigitalTA** [comment]
Grabbed one of these, is the project dead though? I mean I'l do my own shit with it but I won't bother reporting on it if the original author abandoned it.

**Reply by reukiodo** [reply]
The author's site http://pengzhihui.com/ seems to be gone, but archive.org has a redirect to https://pengzhihui.xyz/ which looks to be up and related.

**Reply by reukiodo** [reply]
Which links to https://github.com/peng-zhihui/Project-Quantum and https://gitee.com/coolflyreg163/quark-n though both of those also seem outdated (~3yrs).

**Comment by aaronw2** [comment]
Is there any documentation on the GPIO pins? I have a hardware project and am considering interfacing to this, but I cannot find any documentation on the GPIO pins.

**Reply by reukiodo** [reply]
![20250721_115332.jpg](https://github.com/user-attachments/assets/1376961b-b928-4937-a448-cf581bc8651a)

My expansion board  (Quark-N SoM) came with print, though it doesn't specify front/back, I assume the white is the viewing side and the black is the other (lcd) side.

I still haven't found any documentation for the M.2 pinout of the Quark-N itself.

**Reply by bleaaach** [reply]
> Is there any documentation on the GPIO pins? I have a hardware project and am considering interfacing to this, but I cannot find any documentation on the GPIO pins.

Hi, you can refer to this git repository, which contains the original author's hardware files, including schematics and PCB files. https://github.com/peng-zhihui/Project-Quantum

**Reply by bleaaach** [reply]
> ![20250721_115332.jpg](https://private-user-images.githubusercontent.com/5169351/468763276-1376961b-b928-4937-a448-cf581bc8651a.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMxNDgyNDYsIm5iZiI6MTc1MzE0Nzk0NiwicGF0aCI6Ii81MTY5MzUxLzQ2ODc2MzI3Ni0xMzc2OTYxYi1iOTI4LTQ5MzctYTQ0OC1jZjU4MWJjODY1MWEuanBnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDcyMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA3MjJUMDEzMjI2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDVkMGZmMmNhMjk1NjEzYWZjNTM0MjhhZDA0NjNkOTQ3OGM3ZTM0N2I1NDQ3MTQ4ODM5YTc3NjU5ZTc2YzA4MSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.V184C8iJlATprF-FWx6Wwvj6n-GrhWrwjAzuDYZXL6s)
> 
> My expansion board (Quark-N SoM) came with print, though it doesn't specify front/back, I assume the white is the viewing side and the black is the other (lcd) side.
> 
> I still haven't found any documentation for the M.2 pinout of the Quark-N itself.

Hi, you can refer to this git repository, which contains the original author's hardware files, including schematics and PCB files. https://github.com/peng-zhihui/Project-Quantum
